hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/parser/record/transactionhandler/TokenCreateTransactionHandler.java

```diff
@@ -104,17 +104,18 @@ protected void doUpdateTransaction(Transaction transaction, RecordItem recordIte
         }
 
         var transactionBody = recordItem.getTransactionBody().getTokenCreation();
-        long consensusTimestamp = transaction.getConsensusTimestamp();
+        var consensusTimestamp = transaction.getConsensusTimestamp();
+        var freezeDefault = transactionBody.getFreezeDefault();
```

Comment:
change it back to `boolean`, by our convention, we don't use `var` for primitives

```suggestion
        boolean freezeDefault = transactionBody.getFreezeDefault();
```

Reply:
Good to know. Done.